trade-finance-manager-api/src/v1/helpers/amendment.helpers.js

```diff
@@ -1,27 +1,201 @@
+  // if successful, then updates flag to say email has been sent
+  if (emailResponse) {
+    await managersDecisionUpdateEmailConfirmation(facilityId, amendmentId);
+  }
+};
+
+const emailApprovedWithoutConditionsDeclined = async (amendmentVariables) => {
+  const { user, facilityId, amendmentId } = amendmentVariables;
+  const template = EMAIL_TEMPLATE_IDS.MANUAL_AMENDMENT_DECISION_APPROVED_WO_CONDITIONS_DECLINED;
+
+  const emailResponse = await sendTfmEmail(template, user.email, approvedWithoutConditionsDeclinedDecision(amendmentVariables));
+  // if successful, then updates flag to say email has been sent
+  if (emailResponse) {
+    await managersDecisionUpdateEmailConfirmation(facilityId, amendmentId);
+  }
+};
+
+const emailDeclined = async (amendmentVariables) => {
+  const { user, facilityId, amendmentId } = amendmentVariables;
+  const template = EMAIL_TEMPLATE_IDS.MANUAL_AMENDMENT_DECISION_DECLINED;
+
+  const emailResponse = await sendTfmEmail(template, user.email, declinedDecision(amendmentVariables));
+  // if successful, then updates flag to say email has been sent
+  if (emailResponse) {
+    await managersDecisionUpdateEmailConfirmation(facilityId, amendmentId);
+  }
+};
+
+const sendManualDecisionAmendmentEmail = async (amendmentVariables) => {
+  const { amendment } = amendmentVariables;
+  const { ukefDecision } = amendment;
+
+  try {
+    // if one is approved with conditions and not declined
+    if (isApprovedWithConditions(ukefDecision) && !isDeclined(ukefDecision)) {
+      await emailApprovedWithWithoutConditions(amendmentVariables);
+
+    // if only approved without conditions (and not declined or with conditions)
+    } else if (isApprovedWithoutConditions(ukefDecision) && !isDeclined(ukefDecision) && !isApprovedWithConditions(ukefDecision)) {
+      await emailApprovedWithoutConditions(amendmentVariables);
+
+    // if approved with conditions and declined only
+    } else if (isApprovedWithConditions(ukefDecision) && isDeclined(ukefDecision) && !isApprovedWithoutConditions(ukefDecision)) {
+      await emailApprovedWithConditionsDeclined(amendmentVariables);
+
+    // if approved without condtions and declined only
+    } else if (isApprovedWithoutConditions(ukefDecision) && isDeclined(ukefDecision) && !isApprovedWithConditions(ukefDecision)) {
+      await emailApprovedWithoutConditionsDeclined(amendmentVariables);
+
+    // if declined only
+    } else if (isDeclined(ukefDecision) && !isApprovedWithConditions(ukefDecision) && !isApprovedWithoutConditions(ukefDecision)) {
+      await emailDeclined(amendmentVariables);
+    } else {
+      console.error('Incorrect ukefDecision passed for manual amendment email');
+    }
+  } catch (error) {
+    console.error('Error sending manual amendment underwriter decision email', { error });
+  }
+};
+
+// updates flag if managers decision email sent so not sent again
+const banksDecisionUpdateEmailConfirmation = async (facilityId, amendmentId) => {
```

Comment:
Try-catch block please.

Reply:
was discussing this with @IgnatG .  Please correct if wrong but since it is called in a try catch block in the parent, it shouldn't need one here?

Reply:
agreed. it's already defined inside the other functions.